system-parachains/bridge-hubs/bridge-hub-polkadot/src/lib.rs

```diff
@@ -60,21 +69,29 @@ use frame_system::{
 use pallet_xcm::{EnsureXcm, IsVoiceOfBody};
 pub use sp_consensus_aura::sr25519::AuthorityId as AuraId;
 pub use sp_runtime::{MultiAddress, Perbill, Permill};
-use xcm_config::{FellowshipLocation, GovernanceLocation, XcmOriginToTransactDispatchOrigin};
+#[cfg(not(feature = "runtime-benchmarks"))]
+use xcm_config::XcmRouter;
+use xcm_config::{
+	FellowshipLocation, GovernanceLocation, TreasuryAccount, XcmOriginToTransactDispatchOrigin,
+};
 
 #[cfg(any(feature = "std", test))]
 pub use sp_runtime::BuildStorage;
 
 // Polkadot imports
 use polkadot_runtime_common::{BlockHashCount, SlowAdjustingFeeUpdate};
+use polkadot_runtime_constants::{
+	currency::EXISTENTIAL_DEPOSIT,
+	system_parachain::{ASSET_HUB_ID, BRIDGE_HUB_ID},
+};
```

Comment:
We shouldn't be bringing this in. It's just confusing to have a `SYSTEM_PARA_EXISTENTIAL_DEPOSIT` and an `EXISTENTIAL_DEPOSIT`. Plus, it looks like you just used it because it happens to be equal to something convenient (1 DOT), which we have a type already for (`UNITS`).
```suggestion
use polkadot_runtime_constants::{
	system_parachain::{ASSET_HUB_ID, BRIDGE_HUB_ID},
};
```

Reply:
Agreed, removed in https://github.com/polkadot-fellows/runtimes/pull/130/commits/0c6e184afa829c55e242623fb923defff61dcf2c.